doc/reference/pipeline_spec.md

```diff
@@ -187,6 +187,10 @@ Lines need not end in newline characters.
 `transform.env` is a map from key to value of environment variables that will be
 injected into the container
 
+Note: there are environment variables that are automatically injected into the
```

Comment:
Did you decide not to offer the entire job-info? That's fine, just curious.

Reply:
I'm still planning to do that but I wanted to just do this the quickest way to get it available for customers.